lib/iris/tests/test_pp_module.py

```diff
@@ -213,9 +216,9 @@ def test_wgdos(self):
                                         'nae.20100104-06_0001.pp'))
         r = pp.load(filepath)
 
-        # Check that the result is a generator and convert to a list so that we
-        # can index and get the first one
-        self.assertEqual(type(r), GeneratorType)
+        # Check that the result is an itertools.chain and convert to a list so
+        # that we can index and get the first one.
+        self.assertEqual(type(r), ITERTOOLS_CHAIN_TYPE)
```

Comment:
All rather unpythonic, this kind of testing !
I'd much prefer to see something like ...
```
    self.assertIsInstance(r, collections.Iterable)
    r = list(r)
    self.assertIsInstance(r[0], six.string_types)
```